### PR TITLE
Report preview

### PR DESCRIPTION
**Pull request by adnanjt**

### CODE REVIEW

File: app/views/DR/ReportScreen/index.js [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
 
 export default function ReportScreen({ navigation }) {
-  const [{ answers: {usage} }, setGlobalState] = useContext(context);
+  const [setGlobalState] = useContext(context);
 
   const setSelectedOption = selected => {
     setGlobalState({
-      type: 'ADD_ANSWERS',
-      value: { "usage": selected }
+      type: 'ADD_ANSWERS', //{ answers: {usage} },
```
Comment (inteljudajocu) [inline app/views/DR/ReportScreen/index.js:19]: Remove comments if unnecessary

**Review by inteljudajocu — COMMENTED** [review]
Left comments

File: app/views/DR/Aurora/index.js [hunk 2]
```diff
@@ -1,18 +1,11 @@
-import React, { useEffect } from 'react';
-import { View, TouchableOpacity } from 'react-native';
 import { Text } from 'native-base';
-import { WebView } from 'react-native-webview';
+import React, { useEffect } from 'react';
```
Comment (inteljudajocu) [inline app/views/DR/Aurora/index.js:2]: This file does is not necessary for the report

File: app/views/DR/News.js [hunk 3]
```diff
@@ -7,7 +7,9 @@ import {
   View,
```
Comment (inteljudajocu) [inline app/views/DR/News.js:7]: This file is not necessary for report

**Review by RutsSantos — APPROVED** [review]
LGTM

File: app/views/DR/Aurora/index.js (hunk 2 above)
Comment (RutsSantos) [reply app/views/DR/Aurora/index.js:2]: I think he added other things besides the report

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**